src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +60,252 @@ contract DssSpellCollateralAction {
+    // DIIS Group wallet
+    address constant RWA008_A_MATE             = 0xb9444802F0831A3EB9f90E24EFe5FfA20138d684;
+
+    uint256 constant RWA008_A_INITIAL_DC       = 30_000_000 * RAD;
+    uint256 constant RWA008_A_INITIAL_PRICE    = 30_437_069 * WAD;
+    uint48 constant  RWA008_A_TAU              = 0;
+
+    uint256 constant RWA008_REG_CLASS_RWA      = 3;
+
+    string constant RWA008_DOC                 = "QmdfzY6p5EpkYMN8wcomF2a1GsJbhkPiRQVRYSPfS4NZtB";
+    // -- RWA008 end --
+
+    // -- RWA009 MIP21 components --
+    address constant RWA009                  = 0xd2B101854F64Df738bA601840279838568583F39;
+    address constant MCD_JOIN_RWA009_A       = 0x7122B934F02A15954282Ed41572Ada539864773a;
+    address constant RWA009_A_URN            = 0xD2C8588C72026171Ec3a17369ad0f0734E30915d;
+    address constant RWA009_A_JAR            = 0xa484C16D2Ca15706c4B875710d9e80b7F101572B;
+    // Goerli: CES Goerli Multisig / Mainnet: Genesis
+    address constant RWA009_A_OUTPUT_CONDUIT = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    uint256 constant RWA009_A_INITIAL_DC     = 100_000_000 * RAD;
+    uint256 constant RWA009_A_INITIAL_PRICE  = 100_000_000 * WAD;
+    uint48  constant RWA009_A_TAU            = 0;
+
+    uint256 constant RWA009_REG_CLASS_RWA    = 3;
+
+    string constant RWA009_DOC               = "QmZG31b6iLGGCLGD7ZUn8EDkE9kANPVMcHzEYkvyNWCZpG";
+    // -- RWA009 END --
+
+    function onboardRwa008(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT,
+        address MCD_JOIN_DAI
+    ) internal {
+        // RWA008-A collateral deploy
+
+        bytes32 ilk      = "RWA008-A";
+        uint256 decimals = DSTokenAbstract(RWA008).decimals();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).vat() == MCD_VAT,  "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).ilk() == ilk,      "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).gem() == RWA008,   "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).dec() == decimals, "join-dec-not-match");
+
+        require(RwaUrnLike(RWA008_A_URN).vat() == MCD_VAT,               "urn-vat-not-match");
+        require(RwaUrnLike(RWA008_A_URN).jug() == MCD_JUG,               "urn-jug-not-match");
+        require(RwaUrnLike(RWA008_A_URN).daiJoin() == MCD_JOIN_DAI,      "urn-daijoin-not-match");
+        require(RwaUrnLike(RWA008_A_URN).gemJoin() == MCD_JOIN_RWA008_A, "urn-gemjoin-not-match");
+
+        // Init the RwaLiquidationOracle
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA008_A_INITIAL_PRICE, RWA008_DOC, RWA008_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA008
```

Comment:
I've noticed we are not using the `DssExecLib` functions for `file`s, `rely`s.
We have special functions even for debt ceiling, collateralization ratio, fees.
Any reason for that?
RWA001 in Goerli uses all of them.

Reply:
Tests are based on [this file](https://github.com/makerdao/MIP21-RWA-Example/blob/master/src/RwaSpell.sol) from MIP21-RWA-Example.
I guess we didn't take the time to update it to use `DssExecLib`.
Will do that now.